### ISSUE TITLE
Installation question

### ISSUE THREAD

**Issue by Agusmastex** [ahrm/sioyek#655]
Hello! I'm very excited to give Sioyek a try. I have installed Visual Studio and was making my way to install Qt 5, but the default size  were too large. I read on another issue #322 that Qt 5.14.2 is the needed version to run Sioyek, but the installation size still remains 43.28 GB. What components of Qt 5.14.2 are the minimum required to run Sioyek?

**Comment by ahrm** [comment]
Here are the modules we use: https://github.com/ahrm/sioyek/blob/e44df4f674465ed97e4ce25888bc1fe381d118d8/pdf_viewer_build_config.pro#L9

**Comment by Agusmastex** [comment]
Thanks for the quick response! I am not that tech savvy though, which components do those modules correspond to this checklist?

![image](https://user-images.githubusercontent.com/89489886/223786340-6f1483a5-47d5-4933-a6da-0c230a976875.png)

Thank you for your time!

**Comment by ahrm** [comment]
I am not 100% sure, but one of the MSVCs should be enough.

**Comment by Agusmastex** [comment]
Yes! I am running the commands from an administrator-authorized Visual Studio Developer Command Prompt. That error message was inside that prompt.

**Comment by ahrm** [comment]
The `nmake` should be known to visual studio developer command prompt. My guess is that you have a wrong definition of what visual studio developer command prompt is. It is not the command prompt inside visual studio. It is a separate application which you can launch by searching for `x64 Native Tools Command Prompt` in the start menu.

**Comment by Agusmastex** [comment]
I just realized that there are binaries available for this project. This will better suit my needs. I think my problem was related to #143, when following the steps on [here](https://github.com/ahrm/sioyek/issues/143#issuecomment-1050807646) my error messages changed and I got further into the building process. 

I extend my thanks to you as an open-source software developer. Your efforts in bettering the scientific and open-source community are commendable. Also thank you for assisting me! I am looking forward to the improvements of my scientific workflow sioyek will bring. Goodbye!
